inc/oapv.h

```diff
@@ -37,7 +37,7 @@ extern "C"
 {
 #endif
 #define OAPV_EXPORT
```

Comment:
Do we need this 'define OAPV_EXPORT' still?

Reply:
removed